packages/template-retail-react-app/app/pages/home/index.jsx

```diff
@@ -64,6 +64,8 @@ const Home = () => {
     const {data: productSearchResult, isLoading} = useProductSearch({
         parameters: {
             refine: [`cgid=${HOME_SHOP_PRODUCTS_CATEGORY_ID}`, 'htype=master'],
+            perPricebook: true,
+            allVariationProperties: true,
```

Comment:
@bendvc @joeluong-sfcc for strikethrough price on PLP, we only need these two params. We can add `expand` later when we work on future tickets